pkg/controller/che/che_controller.go

```diff
@@ -528,12 +527,12 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 	}
 
 	// Make sure that CA certificates from all marked config maps are merged into single config map to be propageted to Che components
-	cm, err := deploy.SyncAdditionalCACertsConfigMapToCluster(deployContext)
+	done, err = deploy.SyncAdditionalCACertsConfigMapToCluster(deployContext)
 	if err != nil {
```

Comment:
maybe we can move `err != nil` inside `if !done && !tests { ... }` to have a code with the same manner ?

Reply:
Oh, I see inside we don't use common deploy.Sync , we have some custom logic.